docs/Lint/charp/.editorconfig

```diff
@@ -0,0 +1,39 @@
+# editorconfig.org
```

Comment:
path should be `csharp`* instead of `charp`

Reply:
Good catch! My bad. Fixed.